packages/core/lib/generators/colors_generator.dart

```diff
@@ -15,25 +15,26 @@ import 'generator_helper.dart';
 String generateColors(
   File pubspecFile,
   DartFormatter formatter,
-  FlutterGenColors colors,
+  FlutterGenColors genColors,
 ) {
-  if (colors.inputs.isEmpty) {
+  if (genColors.inputs.isEmpty) {
     throw const InvalidSettingsException(
         'The value of "flutter_gen/colors:" is incorrect.');
   }
 
   final buffer = StringBuffer();
+  final className = genColors.outputs?.className ?? 'FontFamily';
```

Comment:
Why did you use `FontFamily` in color generator?

Reply:
My mistake, I have fixed it.